Lib/test/test_importlib/fixtures.py

```diff
@@ -250,7 +250,7 @@ def build_files(file_defs, prefix=pathlib.Path()):
                 with full_name.open('wb') as f:
                     f.write(contents)
             else:
-                with full_name.open('w', encoding='utf-8') as f:
```

Comment:
These reversions were unintentional as well.

Reply:
I see I ported them to `importlib_metadata` as https://github.com/python/importlib_metadata/commit/6f754c2ae967aa6735f4eee97edbc3da96144bf2, but not merged into the cpython branch.

Reply:
Fixed in 3063cf31d2.